hack/quickstart-metrics.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+#
+# Copyright 2023 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+export TOOLS_IMAGE=quay.io/kuadrant/mgc-tools:latest
+export TMP_DIR=/tmp/mgc
+dockerBinCmd() {
+  echo "docker run --rm -i -u $UID -v ${TMP_DIR}:/tmp/mgc:z -v /config/deploy/local:/config:z --network mgc -e KUBECONFIG=/tmp/mgc/kubeconfig --entrypoint=$1 $TOOLS_IMAGE"
+}
+
+
+export KIND_BIN=kind
+export HELM_BIN=helm
+export KFILT="docker run --rm -i ryane/kfilt"
+export KUSTOMIZE_BIN=$(dockerBinCmd "kustomize")
```

Comment:
There's a bit of duplication here between this and the `quickstart-setup.sh` file. We could move this `dockerBinCmd` function from `quickstart-setup.sh` into a file where it could be sourced by all scripts. We could also maybe create a `.quickstartEnv` file or similar containing exports of all global variables that'll be shared by all quickstart scripts, including `*_BIN` variables, `TMP_DIR` and anything else that will be common to all quickstart scripts.